fieldpath/path.go

```diff
@@ -88,11 +88,11 @@ func MakePath(parts ...interface{}) (Path, error) {
 			fp = append(fp, PathElement{Index: &t})
 		case string:
 			fp = append(fp, PathElement{FieldName: &t})
-		case []value.Field:
-			if len(t) == 0 {
+		case *value.FieldList:
```

Comment:
hrm, can we avoid an allocation if we just store []value.FieldList?

Reply:
I'm not super concerned about allocations on this path since this is only used in test AFAICT. Also, we would need to take the pointer to this anyway?

Reply:
Oh, this is MakePath, sorry. I don't care too much about this, let me find a better place to make a comment.